bokehjs/src/lib/models/layouts/tabs.ts

```diff
@@ -102,6 +112,24 @@ export class TabsView extends LayoutDOMView {
         if (event.target == event.currentTarget)
           this.change_active(i)
       })
+      const tooltip = this.tooltips[i]
+      if (tooltip) {
+        tooltip.useRelativeBoundingBox = false
+        tooltip?.model.setv({
+          target: el,
+        })
+        const toggleTooltip = (visible: boolean) => {
+          tooltip.model.setv({
+            visible,
+          })
```

Comment:
Use this instead:
```suggestion
          tooltip.model.visible = visible
```
`setv()` doesn't currently allow for statically typed arguments and should be used only when really needed (e.g. simultaneously setting multiple properties).